CMakeLists.txt

```diff
@@ -166,7 +168,11 @@ if("virtual" IN_LIST COMPILE_TARGETS)
   add_dependencies(ccf ccf.virtual)
 endif()
 
-install(EXPORT ccf DESTINATION cmake)
+install(
+  EXPORT ccf
+  DESTINATION cmake
+  FILE ccf-targets.cmake
```

Comment:
Exporting that as "ccf.cmake" caused a double include, and loads of unpleasant errors.